### PR TITLE
Remove references of mercator tool

### PR DESCRIPTION
**Pull request by shreekarSS**
Signed-off-by: Shreekara <sshreeka@redhat.com>

## Related Issues and Dependencies
Fixes: https://github.com/thoth-station/package-extract/issues/485
…

## Description
Remove all references of mercator from package-extract repo

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -516,26 +515,6 @@ To extract packages present on the resulting image run:
...
-          "classifier": "Development Status :: 5 - Production/Stable\nIntended Audience :: Developers\nLicense :: OSI Approved :: MIT License\nLicense :: OSI Approved :: Python Software Foundation License\nOperating System :: OS Independent\nProgramming Language :: Python\nProgramming Language :: Python :: 2Programming Language :: Python :: 2.6\nProgramming Language :: Python :: 2.7\nProgramming Language :: Python :: 3\nProgramming Language :: Python :: 3.3\nProgramming Language :: Python :: 3.4Topic :: Software Development :: Libraries :: Python Modules",
-          "description": "iniparse is an INI parser for  Python which is API compatible\nwith the standard library's ConfigParser, preserves structure of INI\nfiles (order of sections & options, indentation, comments, and blank\nlines are preserved when data is updated), and is more convenient to\nuse.",
-          "home-page": "http://code.google.com/p/iniparse/",
-          "license": "MIT",
-          "name": "iniparse",
-          "platform": "UNKNOWN",
-          "summary": "Accessing and Modifying INI files",
-          "version": "0.4"
-        }
-      },
```
Comment (mayaCostantini) [inline README.rst:538]: I think the `"mercator"` field goes until ~L810, could you remove the rest of it from this example?

**Review by mayaCostantini — COMMENTED** [review]
lgtm, just a few comments :+1: 
Have you tested extracting dependencies from an image with those changes (with the CLI)?

File: CHANGELOG.md [hunk 2]
```diff
@@ -168,7 +168,6 @@
 * Use credentials to communicate with registry if needed
 * Quote user input
 * The items field can be None
-* Mercator can output None
```
[the remark this answers is not in the document]
Comment (shreekarSS) [reply CHANGELOG.md:171]: ack

**Review by mayaCostantini — APPROVED** [review]
Very good :100: Let's have this in

**Review by harshad16 — CHANGES_REQUESTED** [review]
please include `kubernetes~=12.0.1` in the requirements.txt to fix build issue.
/hold

**Comment by harshad16** [comment]
Please rebase as well
keep the pr to one commit, if possible.

**Comment by shreekarSS** [comment]
> please include `kubernetes~=12.0.1` in the requirements.txt to fix build issue. /hold

Thanks , fixed

**Pull request merged**